notification-rule-config-service-api/src/main/proto/org/hypertrace/notification/config/service/v1/notification_rule_config_service.proto

```diff
@@ -11,6 +11,7 @@ service NotificationRuleConfigService {
   rpc UpdateNotificationRule(UpdateNotificationRuleRequest) returns (UpdateNotificationRuleResponse) {}
   rpc GetAllNotificationRules(GetAllNotificationRulesRequest) returns (GetAllNotificationRulesResponse) {}
   rpc DeleteNotificationRule(DeleteNotificationRuleRequest) returns (DeleteNotificationRuleResponse) {}
+  rpc GetNotificationRuleById(GetNotificationRuleByIdRequest) returns (GetNotificationRuleByIdResponse) {}
```

Comment:
almost all of our CRUD services use the convention of `GetXyz` - the by ID is implied with a singular get.

Reply:
ah, it was easier to change the api name after the build failure 😅
fixing